### ISSUE TITLE
Question about the initialization of prototypes

### ISSUE THREAD

**Issue by Hugo-cell111** [WangChangqi98/CSS#4]
Hi! I find that the prototypes are initialized as the torch.zeros Tensors, and I wonder whether the noise will be brought by the representation space pseudo labels? Thanks!

**Comment by WangChangqi98** [comment]
Hi, the first prototype in the training process is the prototype calculated in the first mini-batch, you can find it in ./generalframeworks/loss/loss.py, and there will be some noise in pseudo-labels from the representation space, we follows prior works and use a simple threshold sampling strategy to improve the quality of the pseudo-labels and meanwhile, many other approaches can be used to polish the pseudo-labels from the representation space.

**Comment by Hugo-cell111** [comment]
I see. Thank you!
